tf_files/aws/modules/common-logging/variables.tf

```diff
@@ -40,3 +40,18 @@ variable "threshold"{
 variable "slack_webhook"{
   default = ""
 }
+
+variable "log_dna_function"{
+  description = "Lambda function ARN for logDNA"
+  default     = ""
```

Comment:
what happens if I set this variable to empty string `""` ?  will the lambda be smart enough to skip trying to invoke a lambda?  It looks like the lambda function checks for `None` instead of an `arn` or whatever?

Reply:
validation added in that latest commit